src/gui/TimeLineWidget.cpp

```diff
@@ -417,18 +357,25 @@ void TimeLineWidget::mousePressEvent( QMouseEvent* event )
 void TimeLineWidget::mouseMoveEvent( QMouseEvent* event )
 {
 	parentWidget()->update(); // essential for widgets that this timeline had taken their mouse move event from.
-	const TimePos t = m_begin + static_cast<int>( qMax( event->x() - m_xOffset - m_moveXOff, 0 ) * TimePos::ticksPerBar() / m_ppb );
-	// Fine adjust when both ctrl and shift are held, hide ctrl+shift hint
-	bool unquantized = event->modifiers() == (Qt::ControlModifier | Qt::ShiftModifier);
+	// Fine adjust when ctrl is held, hide ctrl hint
+	bool unquantized = event->modifiers() & Qt::ControlModifier;
```

Comment:
This forces all ctrl bound actions to be unquantized

Reply:
How should one unquantize a ctrl action if ctrl is the unquantize key? Or should all ctrl actions always be unquantized?

Reply:
This is why I used ctrl+shift as the unquantize modifier. If base actions are only allowed to bind to ctrl or shift (not both) then the combination is free to use for fine adjust.

Reply:
Ok, I see your point. Let's do it your way, it makes no difference atm anyway. Tho I must say I'm opposed giving special meaning to the order of modifier key press. I can think of no other example where this is a thing. How would you even write it in a shortcut table? Ctrl+LMB+Shift vs Shift+LMB+Ctrl vs Ctrl+Shift+LMB... It's starting to sound like Emacs.

Reply:
I think of toggling quantize as a separate action from beginning an action, so I didn't really interpret it as an order-specific thing. IIRC (at least in the existing PR) releasing ctrl during a shift action reverts to quantized, and vice versa for a ctrl action. Regardless, it felt dirtier to me that certain bindings could only be un/quantized than the admittedly unusual "press opposite modifier to unquantize" method.

Personally I'm hesitant about differentiating click from drag, but clearly there are a lot of actions we want to fit into this one area 😅

Reply:
Ah I see now you used Ctrl+RMB as "move closest". In that case it makes perfect sence (I had removed that feature in this PR, as I found it overkill) ... But with that logic a user might press ctrl+shift+RMB thinking it would be "move closest unquantized" and instead it becomes "move right unquantized", due to shift+RMB being "move right". I say, keep it simple. Yes drag/click is a bit wonky too, but at least the user's own intention with those actions are more distinguishable.